### ISSUE TITLE
"self-update" threw an fatal first time, now ends with "Catchable fatal error" every time

### ISSUE THREAD

**Issue by damienmckenna** [hechoendrupal/DrupalConsole#1442]
The first time I ran "drupal self-update" today (about two minutes ago, right after the tweet announcement of v0.9.9, the following error:

```
$ drupal self-update
The console has been updated to the latest version.
e Symfony\Component\Console\Input\InputArgument;
use Symfony\Component\Console\Input\InputInterface;
use Symfony\Component\Console\Output\BufferedOutput;
use Symfony\Component\Console\Output\OutputInterface;
use Symfony\Component\Console\Application;
use Symfony\Component\Console\Helper\HelperSet;

class Command
{
private $application;
private $name;
private $processTitle;
private $aliases = array();
private $definition;
private $help;
private $description;
private $ignoreValidationErrors = false;
private $applicationDefinitionMerged = false;
private $applicationDefinitionMergedWithArgs = false;
private $code;
private $sy
Fatal error: Class 'Symfony\Component\Console\Event\ConsoleTerminateEvent' not found in phar:///Users/dmckenna/.composer/vendor/drupal/console/bin/console/vendor/symfony/console/Application.php on line 902
```

Now when I run the command again it leaves the following:

```
$ drupal self-update

[WARNING] The latest version 0.9.9, was already installed on your system.                                              

Catchable fatal error: Argument 1 passed to Symfony\Component\Console\Application::setDispatcher() must implement interface Symfony\Component\EventDispatcher\EventDispatcherInterface, null given, called in phar:///Users/dmckenna/.composer/vendor/drupal/console/bin/console/src/Command/SelfUpdateCommand.php on line 63 and defined in phar:///Users/dmckenna/.composer/vendor/drupal/console/bin/console/vendor/symfony/console/Application.php on line 92
```

**Comment by jmolivas** [comment]
Seems like we need to get back to the ugly exit hack.

**Comment by geerlingguy** [comment]
I'm hitting this as well, now.

**Comment by geerlingguy** [comment]
See, in particular: https://travis-ci.org/geerlingguy/ansible-role-drupal-console/jobs/95685471#L670

**Comment by jmolivas** [comment]
Fixed with #1453

**Comment by jmolivas** [comment]
Fix will be included on the v0.9.10 release.
